lib/pakyow/command_line_interface.rb

```diff
@@ -0,0 +1,61 @@
+require "thor"
```

Comment:
Yes, seems Thor automatically matches commands generously. So it'll actually do anything from `s` to `server`. Not sure how it'd behave with two commands starting with "s". I'm having trouble verifying it in their documentation anywhere, but that's the behavior I noted. It'd be great to have these things under test for clearer specification.